src/app/sign-in-modal/sign-in-modal.component.html

```diff
@@ -62,27 +66,32 @@ <h3>Or register if you are new here</h3>
           <label for="registerEmail">Email</label>
         </span>
 
-        <span class="p-float-label">
-          <input
-            type="password"
-            pInputText
+        <p-floatLabel class="p-float-label">
+          <p-password
             id="registerPassword"
+            [toggleMask]="true"
             placeholder="Should have at least 8 characters"
+            promptLabel="Choose a password"
+            weakLabel="Don`t be a fool"
+            mediumLabel="Not bad"
+            strongLabel="Excellent"
```

Comment:
for such repeating messages it would be nice to create some ts file with them and reuse. 
texts = {
  validationErrors: {
    passwordMediumLavel: 'Not bad'
  }
}